test/e2e/framework/util.go

```diff
@@ -163,6 +163,9 @@ const (
 
 	// Number of times we want to retry Updates in case of conflict
 	UpdateRetries = 5
+
+	// number of objects that gc can delete in a second
+	gcThroughput = 15
```

Comment:
Not sure where "15" comes from. GC throughput is QPS/2 since it sends 2 requests to delete an object. The maths worked last time I checked the 500-gce density test :)